packages/anoncreds/src/formats/LegacyIndyCredentialFormatService.ts

```diff
@@ -300,23 +300,12 @@ export class LegacyIndyCredentialFormatService implements CredentialFormatServic
     const credentialRequest = requestAttachment.getDataAsJson<AnonCredsCredentialRequest>()
     if (!credentialRequest) throw new CredoError('Missing indy credential request in createCredential')
 
-    const { credential, credentialRevocationId } = await anonCredsIssuerService.createCredential(agentContext, {
+    const { credential } = await anonCredsIssuerService.createCredential(agentContext, {
       credentialOffer,
       credentialRequest,
       credentialValues: convertAttributesToCredentialValues(credentialAttributes),
     })
 
-    if (credential.rev_reg_id) {
```

Comment:
I removed it from here because we don't actually support revocation with legacy indy credentials (but we do in AnonCreds, so tags are added to issued credentials there)

Reply:
WDYM? A credential with legacy identifiers can't use revocation? Is that because it's not implemented in the legacy credential format service?

Reply:
Yes, we do not currently support issuing revocable legacy indy credentials, so I don't see how this if statement can be true.

Reply:
what do you mean by "legacy indy credentials"? is there a short documentation about what is considered legacy?

Reply:
@genaris I know this is already merged but could I get a bit of clarification here; is this method only for *issuing credentials*? The name `acceptRequest` implies it could be accepting a credential issued by another agent that does support issuing legacy creds, no? Currently our ecosystem uses revocation with credentials that don't start with `did:indy` etc. ie. "legacy" credentials as I understand it, so it seems like we would need to keep this `if` block. Let me know what you think

Reply:
The naming here is mainly based on DIDComm Issue Credential protocols, where the steps are propose -> offer -> request -> issue (see [here](https://github.com/hyperledger/aries-rfcs/blob/main/features/0453-issue-credential-v2/README.md#messages)). This `acceptRequest` is about responding to the `request-credential` message, sent from the holder to the issuer to ask for the actual credential issuance.

So this method is only called in issuer use case. As you can see some lines before, we are calling `anoncredsIssuerService.createCredential` without any revocation-related `CreateCredentialOptions` parameter (revocationRegistryDefinitionId, revocationRegistryIndex, revocationStatusList), like we do in [AnonCredsCredentialFormatService](https://github.com/openwallet-foundation/credo-ts/blob/dcb363dc7e7abbbcce8b3a344a487a186eb5c88e/packages/anoncreds/src/formats/AnonCredsCredentialFormatService.ts#L344).

Reply:
Ah, ok. Got it. Thank you!